downstream/modules/platform/con-eda-2-5-with-controller-2-4.adoc

```diff
@@ -0,0 +1,59 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-07-05
+
+:_mod-docs-content-type: CONCEPT
+
+[id="eda-2-5-with-controller-2-4_{context}"]
+= {EDAName} 2.5 with controller 2.4
+
+Use the following example to populate the inventory file to deploy a new single instance of {EDAName} 2.5 with controller 2.4. For {EDAName}, the requirements for a connection to controller are the `automation_controller_main_url` pointing to the 2.4 controller URL and `username/password`.
+
+
+----
+[automationedacontroller]
+
+[automationgateway]
+
```

Comment:
We also need a hostname here, but since we're doing a single EDA installation, the hostname you use for line 14 should be the same host here on line 16. That way EDA and gateway will install on the same host.